dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe

```diff
@@ -46,6 +45,14 @@ INSTALL()
 {
 	LUAVERSION=$(lua -v | sed 's/Lua //' | head -c3)
 	make LUA_LIBDIR="$libDir/lua/$LUAVERSION" install
+
+#	copy the doc files
+#	mkdir -p $docDir
+#	cp -r doc/* $docDir
+
+#	copy the header file
+#	mkdir -p $includeDir
+#	cp src/lfs.h $includeDir
```

Comment:
Is it intended to let these lines commented?

Reply:
Aside from wordgrinder I haven't found any recipe looking for the header file, so commenting them out for now